### DISCUSSION TITLE
Changes screen focus via mouse hover

### DISCUSSION THREAD

**Discussion by 42ajakob** [qtile/qtile#4339] [Ideas]
Make it possible for users to not have the need to click when hovering to another Screen to change the focus.
So that all we have to do is hover over the screen to focus it.

**Comment by elParaguayo** [comment]
I don't think this needs to be closed. It's a reasonable request and similar questions have been asked before.

It's probably not high on the team's list of priorities but might be something nice to implement.

**Reply by 42ajakob** [reply]
I agree just having a function that detects mouse hovering could be nice for some people for various reasons. 
I just think there are many many more things to take care of so I closed it.

Was pretty used to i3 where you could just move the pointer to a different screen and then open a implementation with rofi or systemd on this screen. Or even change focus when you swap the workspace. Tho I didn't take a closer look into it if this is already possible with the current qtile version.

**Comment by zordsdavini** [comment]
Doesn't `follow_mouse_focus = True` in config do that?

**Reply by elParaguayo** [reply]
Not for hovering over an empty screen.
